scripts/synthetix/facilicate_trade.js

```diff
@@ -0,0 +1,88 @@
+const { getExchange } = require("../utils/trading_strategy_helpers")(web3, artifacts)
+const { SynthetixJs } = require("synthetix-js")
+const { calculateBuyAndSellAmountsFromPrice, fetchTokenInfoFromExchange } = require("../utils/trading_strategy_helpers")(
+  web3,
+  artifacts
+)
+
+const sUSDAddress = {
+  mainnet: "0xAe38b81459d74A8C16eAa968c792207603D84480",
+  rinkeby: "0x1b642a124CDFa1E5835276A6ddAA6CFC4B35d52c",
+}
+
+const sETHAddress = {
+  mainnet: "0xD0DC005d31C2979CC0d38718e23c82D1A50004C0",
+  rinkeby: "0x0647b2C7a2a818276154b0fC79557F512B165bc1",
+}
+
+module.exports = async callback => {
+  try {
+    const snxjs = new SynthetixJs({ networkId: await web3.eth.net.getId() })
+    const exchange = await getExchange(web3)
+    const defaultAccount = (await web3.eth.getAccounts())[0]
+
+    // Both of these hardcoded tokens are assumed to have 18 decimal places.
+    // We "trust" that this will always be the case although it seems
+    // that synthetix reserves the authority to upgrade their token
+    // This could mean issuing a new one with a different number of decimals.
+    const sETHKey = await snxjs.sETH.currencyKey()
+    const sUSDKey = await snxjs.sUSD.currencyKey()
+
+    // Fetch token IDs and other relevant token information for order placement.
+    const referenceTokenId = (await exchange.tokenAddressToIdMap.call(sUSDAddress[process.env.NETWORK_NAME])).toNumber()
+    const etherTokenId = (await exchange.tokenAddressToIdMap.call(sETHAddress[process.env.NETWORK_NAME])).toNumber()
+
+    const tokenInfoPromises = fetchTokenInfoFromExchange(exchange, [referenceTokenId, etherTokenId])
+    const stableToken = await tokenInfoPromises[referenceTokenId]
+    const targetToken = await tokenInfoPromises[etherTokenId]
```

Comment:
This is only needed to get the decimal. I'd say we can save these two calls and just hardcode this to 18 decimals (the way they compute their exchange rates seem to only work for tokens with 18 decimals).

This would become even more easy if we read token information from the tokenList.json

Reply:
Have decided against this because `tokenList` is awkward to access for our purposes here.